docs/user/getting_started.rst

```diff
@@ -23,7 +23,19 @@ easily be installed through anaconda.
 Installation
 ------------
 
-Installing CUQIpy is easy. Open your terminal (Linux and Mac) or Anaconda Prompt (Windows) and install it using pip:
+Before installing CUQIpy, We highly recommend creating a new Python environment using conda. This will ensure that CUQIpy's dependencies do not interfere with other Python packages you may have installed. Additinally, we also install pip in the environment.
+
+.. code-block:: sh
+
+   conda create -n cuqipy-env pip
```

Comment:
I suggest we call the environment "cuqipy" and not "cuqipy-env". We can explain a bit in the text if needed.

Furthermore, often an issue when working like this is that "ipykernel" must also be installed to run notebooks etc. Thus I would suggest to include a description on that also if we go down the road to include environments.

Reply:
Thanks for the suggestion. I have added the following sentence accordingly:
> It is also suggested to install ipykernel as we usually work with jupyter notebooks a lot:
pip install ipykernel

Reply:
Updated to `cuqipy-env` after stand-up discussion